analyticConfigurations/computed_metric-aws.tag.json

```diff
@@ -0,0 +1,23 @@
+{
+  "analyticConfiguration" : {
+    "tenantId" : null,
+    "type" : "COMPUTED_METRIC",
+    "id" : null,
+    "packageId" : null,
+    "name" : "AWS Target Group",
+    "scope" : {
+      "elementName" : null,
+      "elementType" : "Target Group",
+      "elementTags" : null,
+      "metricMatches" : null
+    },
+    "metrics" : [ {
+        "match": "netuitive.http.successpercent",
```

Comment:
Can you name it netuitive.aws.applicationelb.httpsuccesspercent to keep with our naming conventions?